eng/config/PublishData.json

```diff
@@ -253,6 +253,16 @@
       "insertionTitlePrefix": "[d17.14 P1]",
       "insertionCreateDraftPR": false
     },
+    "release/dev18.0": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "vsBranch": "dev/monicaro/versioning",
```

Comment:
I assume this is temporary until we get an actual branch

Reply:
correct, that's in progress